tests/conftest.py

```diff
@@ -239,6 +251,54 @@ def testing_files_for_process(cleanup_backups):
     },
 ]
 
+MIGRATED_MONGO_TESTING_SAMPLES = [
```

Comment:
I think this name is too vague: there are lots of migrations, perhaps `MONGO_TESTING_SAMPLES_WITH_FILTERED_POSITIVE_FIELDS`?